setup.py

```diff
@@ -17,7 +17,7 @@
     author="Research Computing Service, Imperial College London",
     author_email="rcs-support@imperial.ac.uk",
     url="https://github.com/ImperialCollegeLondon/R2T2",
-    install_requires=["wrapt"],
+    install_requires=["wrapt", "bibtexparser"],
```

Comment:
Have we not moved to poetry? Should this not go in pyproject.toml?

Reply:
It should, but #67 needs to be merged first.